### PR TITLE
#3 -feature/student-profile-section-saadia

### LINKED ISSUE #3: Student Dashboard - Profile section
## Story

- **As a** student

- **I Want** be able to see my information in the dashboard

- **So That** I can update it

## Acceptance Criteria

1. Given I clicked on the login button in the login page
   - When I'm redirected to my dashboard
   - Then I see a welcome message and  a profile section with picture , bio, mentor name and edit button
2. Given I'm in the profile section
   - When I click on the 'edit profile' button 
   - Then the button is hidden and the 'save' and 'cancel' buttons are shown and  the bio and the picture become editable
5. Given I edited the bio or uploaded a new image
   - When I click on the 'save' button 
   - Then my changes are saved
6. Given I edited the bio or uploaded a new image
   - When I click on the 'cancel' button 
   - Then my changes are not saved
   
## Implementation Notes

1. How to upload img ?
2. Placeholder img

## Checklist:

- [ ] I have defined the user story
- [ ] I have defined some acceptance criteria
- [ ] I have included any implementation notes
- [ ] I have included all relevant links

### PR DESCRIPTION
**Pull request by SaadiaELF**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Create a profile section front-end, where the bio is editable and a profile picture can be changed 
I used Material UI library for styling and creating components and for uploading images using  `react-uploader`

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #3 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/components/Profile.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+					{/* Showing the upload button and making the bio editable only when we click on edit profile  */}
+					{editable ? (
+						<Stack>
+							<UploadButton
+								uploader={uploader}
+								options={{
+									maxFileCount: 1,
+									maxFileSizeBytes: 5000000,
+									mimeTypes: ["image/jpeg", "image/png"],
+								}}
```
Comment (Arbeits-Sachen) [inline client/src/components/Profile.js:69]: Nice, I didnt know you could do that.

**Review by Arbeits-Sachen — APPROVED** [review]
Amazing Code!

**Comment by mrvicthor** [comment]
Good job on the student profile page

**Review by mrvicthor — APPROVED** [review]
Well done Saadia and the team.

**Pull request merged**